src/components/Pagination.tsx

```diff
@@ -0,0 +1,72 @@
+import { Dispatch, SetStateAction } from "react";
+import { IconButton } from "src/components";
+import { Css, Palette } from "src/Css";
+import { SelectField } from "src/inputs";
+import { useTestIds } from "src/utils";
+
+export type PageSettings = {
+  page: number;
+  perPage: number;
+};
+
+type PaginationProps = {
+  label: string;
+  hasNextPage: boolean;
+  settings: PageSettings;
+  setSettings: Dispatch<SetStateAction<PageSettings>>;
+};
+
+export function Pagination(props: PaginationProps) {
+  const { hasNextPage, label, settings, setSettings } = props;
+  const { perPage } = settings;
+  const pageOptions = Array(5)
+    .fill(0)
+    .map((_, i) => ({ id: (i + 1) * 50, name: ((i + 1) * 50).toString() }));
```

Comment:
Suggestion: We should stop hard-coding this before we land this in Beam.

I believe we probably need like `PageSettings.totalCount`, and then this page options is basically `totalCount / perPage` --> list of pages they can choose in the drop down.

cc @roberthdg does that sound right?

Reply:
This `pageOptions` would be the number of items that the user could select to appear per page, not the actual page index (could be renamed to better match the functionality)

![image](https://user-images.githubusercontent.com/24662091/219088184-7b651abb-cf5b-43b5-a80d-613d1d82e7b5.png)

Reply:
Oh jeez, that makes sense. Thanks @roberthdg !

@allangaldinosilva since this doesn't depend on any props, could you make it a top-level `const` for now and use an in-source comment like `// Make a list of 50/100/150/etc page sizes for the user to chose`.

Reply:
Done 👍 thanks guys!